apps/ledger-live-desktop/src/renderer/families/cosmos/DelegationFlowModal/Info/index.jsx

```diff
@@ -53,7 +53,7 @@ export default function CosmosEarnRewardsInfoModal({ name, account, parentAccoun
       footerLeft={
         <LinkWithExternalIcon
           label={t("delegation.howItWorks")}
-          onClick={onLearnMore(account.currency.id)}
+          onClick={() => onLearnMore(account.currency.id)}
```

Comment:
Interesting - why did this cause 3 tabs to appear?

Reply:
It's because react would have re-rendered this component 3 times, and every time it renders it calls `onLearnMore` which opens the link.

Reply:
I know it is not this particular bug, but do we need to add this fix to any other family?

Reply:
I've had a quick look and I can't find any other `onClick` props that need changing